### DISCUSSION TITLE
My friend's git account has been hacked and the mail id and password has been changed by the hacker

### DISCUSSION THREAD

**Discussion by Kachroo-cheena** [community/community#41772] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

First the mail id was hacked and through which the hacker has hacked the git and changed the mail id to his. After restoring the mail id ,he wasnt able to access his git account due to the change in the mail id and password. He has been struggling since months as there are many confidential codes.

Kindly help!

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Kachroo-cheena ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :) The Community is unable to assist in situations like this so I'm going to lock this post.
